src/rvw.cpp

```diff
@@ -226,18 +227,19 @@ void vwtrain(Rcpp::List & vwmodel, SEXP & data=R_NilValue, Rcpp::Nullable<Rcpp::
 //'vwtrain(test_vwmodel, data = ext_train_data)
 //'vwtrain(test_vwmodel, data = ext_test_data)
 // [[Rcpp::export]]
-Rcpp::NumericVector vwtest(Rcpp::List & vwmodel, SEXP & data=R_NilValue, std::string probs_path = "", Rcpp::Nullable<Rcpp::String> readable_model=R_NilValue, bool quiet=false,
-                           SEXP & namespaces=R_NilValue, SEXP & keep_space=R_NilValue,
-                           SEXP & targets=R_NilValue, SEXP & probabilities=R_NilValue,
-                           SEXP & weight=R_NilValue, SEXP & base=R_NilValue, SEXP & tag=R_NilValue, SEXP & multiline=R_NilValue) {
+Rcpp::NumericVector vwtest(Rcpp::List & vwmodel, SEXP data, std::string probs_path = "", Rcpp::Nullable<Rcpp::String> readable_model=R_NilValue, bool quiet=false,
```

Comment:
Missing typing for `SEXP data`

Reply:
Actually, I wanted to use `SEXP` here because `data` can be `data.frame` or `String`.